Script/AtomicNET/AtomicNET/Scene/CSComponentCore.cs

```diff
@@ -332,6 +332,21 @@ void HandleComponentLoad(uint eventType, ScriptVariantMap eventData)
 
         }
 
+        void HandleComponentAssemblyReference(uint eventType, ScriptVariantMap eventData)
+        {
+#if ATOMIC_DESKTOP || ATOMIC_ANDROID
+            string assemblyPath = eventData["AssemblyPath"];
```

Comment:
Probably don't need this on Android?  Can leave it if it isn't hurting anything...